extensions/ns-discount/src/PriceList.tsx

```diff
@@ -0,0 +1,49 @@
+import { List } from '@raycast/api'
+import { useEffect, useState } from 'react'
+import { fetchGameDetail } from './api'
+import { IGame, IPrice } from './model'
+import { calcCutoff } from './utils'
+
+export default function PriceList({ game }: { game: IGame }) {
+  const [prices, setPrices] = useState<IPrice[]>([])
+  const [showPrices, setShowPrices] = useState<IPrice[]>([])
+
+  useEffect(() => {
+    try {
+      prices.length === 0 &&
+        fetchGameDetail(game.appid).then((res) => {
+          setPrices(res.prices)
+          setShowPrices(res.prices)
+        })
+    } catch (error) {
+      throw new Error('Failed to fetch game detail')
+    }
+  })
+
+  const onSearchTextChanged = (text: string) => {
+    if (text.length === 0) {
+      setShowPrices(prices)
+    } else {
+      setShowPrices(prices.filter((price) => price.country.includes(text)))
+    }
+  }
+
+  return (
+    <List onSearchTextChange={onSearchTextChanged}>
+      <List.Item key="date" title="截止日期" accessories={[{ text: game.leftDiscount.replace('天', ' 天') }]} />
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English, please keep it in English 😄